integration/sync_test.go

```diff
@@ -226,8 +227,9 @@ func (s *SyncSuite) TestYamlUntagged(c *check.C) {
 	defer os.RemoveAll(tmpDir)
 	dir1 := path.Join(tmpDir, "dir1")
 
-	image := "alpine"
-	imageRef, err := docker.ParseReference(fmt.Sprintf("//%s", image))
+	registry := "k8s.gcr.io"
+	image := "pause"
+	imageRef, err := docker.ParseReference(fmt.Sprintf("//%s/%s", registry, image))
```

Comment:
(I’d be a tiny bit happier with avoiding this bit of string formatting, i.e. keep `image := "k8s.gcr.io/pause"`, and use `reference.Domain(imageRef.DockerReference()), imagePath` for creating the YAML. That also makes the two YAML files generated completely consistently.

But this works just fine, feel free to leave as is.)